api/src/main/java/com/codeforcommunity/dto/school/School.java

```diff
@@ -1,19 +1,40 @@
 package com.codeforcommunity.dto.school;
 
 import com.codeforcommunity.enums.Country;
+import java.util.ArrayList;
+import java.util.List;
 
 public class School {
 
   private Long id;
   private String name;
   private String address;
   private Country country;
+  private Boolean hidden;
+  private List<SchoolContact> contacts;
 
-  public School(Long id, String name, String address, Country country) {
+  public School(Long id, String name, String address, Country country, Boolean hidden) {
     this.id = id;
     this.name = name;
     this.address = address;
     this.country = country;
+    this.hidden = hidden;
+    this.contacts = new ArrayList<SchoolContact>();
+  }
+
+  public School(
+      Long id,
+      String name,
+      String address,
+      Country country,
+      Boolean hidden,
+      List<SchoolContact> contacts) {
+    this.id = id;
+    this.name = name;
+    this.address = address;
+    this.country = country;
+    this.hidden = hidden;
+    this.contacts = contacts;
```

Comment:
Ahh good note--definitely something we should watch out for in the future. In most instances I would agree, but I don't anticipate this being an issue here, only because this class is strictly used as a DTO--meaning we won't be using instances of this class for any actual logic processing, but just to deserialize API requests and serialize API responses.

Reply:
@jackblanc do you have any thoughts on this?